docs/data/toolpad/core/components/sign-in-page/ComponentsPropsSignIn.js

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { SignInPage } from '@toolpad/core';
+
+const providers = [{ id: 'credentials', name: 'Email and Password' }];
+
+export default function ComponentsPropsSignIn() {
+  return (
+    <SignInPage
+      signIn={(provider) => alert(`Signing in with "${provider.name}"`)}
+      componentProps={{
+        email: { autoFocus: false, variant: 'standard' },
+        password: { variant: 'standard' },
+        button: { variant: 'outlined' },
```

Comment:
`button` seems a bit of a generic name, maybe `submitButton` or something? Also maybe `emailField`, `passwordField`...